libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -605,6 +605,8 @@ def _combine_llm_outputs(self, llm_outputs: List[Optional[dict]]) -> dict:
             token_usage = output["token_usage"]
             if token_usage is not None:
                 for k, v in token_usage.items():
+                    if v is None:
```

Comment:
what would happen if `v` were a string due to a bad API response? Instead of checking what it is, should we be checking what it's not? is `v` expected to be an `int` usually?

Reply:
I believe `v` is already checked inside _update_token_usage(). `v` can be a list or a dict or an int. However we don't check for it being a None value which breaks the code when we try to add up None + None

Reply:
do you know what it looks like when it's a list?